andela/banshee-bn-backend
# #167891563 Setup ESLint

#### What does this PR do?
Setup eslint configuration
#### Description of Task to be completed?
* Install eslint, eslint-config-airbnb-base and eslint-plugin-import packages
* Add eslint configuration file
#### How should this be manually tested?
* Install vscode [eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) extension 
* Go to ```index.js``` file in the ```src``` directory and declare a random variable
* You should see a ```variable is declared but it's value is never read``` error
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#167891563](https://www.pivotaltracker.com/story/show/167891563)
#### Screenshots (if appropriate)
N/A
